ehcache-core/src/main/java/org/ehcache/core/spi/store/tiering/CachingTier.java

```diff
@@ -107,6 +108,8 @@ public interface CachingTier<K, V> extends ConfigurationChangeSupport {
    */
   void setInvalidationListener(InvalidationListener<K, V> invalidationListener);
 
+    Map<K, Store.ValueHolder<V>> bulkGetOrComputeIfAbsent(Set<? extends K> keys, Function<Set<? extends K>, Iterable<? extends Map.Entry<? extends K, ? extends Store.ValueHolder<V>>>> mappingFunction) throws StoreAccessException;
```

Comment:
I think we can relax the `keys` type here to `Iterable<? extends K>`. We should also add some javadoc.

Reply:
Relaxed the key type from `Set<? extends K>` to `Iterable<? extends K>`
Added method level javadoc as well